:bug: fix unsupported packageJson name for read-pkg-up

MarkLyck opened the pull request:
fixes node preset for newer versions of read-pkg-up "pkg" -> "packageJson" (with backwards compatibility)

fixes #205 
closes #192

In later versions of https://github.com/sindresorhus/read-pkg-up, the package.json name changed from `pkg` to `packageJson`

This PR adds support for that change while still being backwards compatible.

I tested this in my own repository and gitmoji-changelog generates fine after this change.

Feel free to test this preset on my public repository: https://github.com/MarkLyck/test-pw

frinyvonnick commented:
Hi @MarkLyck 👋 

Thank you for opening this pull request. As I said [here](https://github.com/frinyvonnick/gitmoji-changelog/pull/192#issuecomment-747306175), I would prefer directly updating the package. Can you do it in this PR please?

MarkLyck commented:
> Hi @MarkLyck 👋
> 
> Thank you for opening this pull request. As I said [here](https://github.com/frinyvonnick/gitmoji-changelog/pull/192#issuecomment-747306175), I would prefer directly updating the package. Can you do it in this PR please?

Oh I did not see that PR 🤦‍♂️

I don't actually see `read-pkg-up` any where in the package.json file? Do you know where this is located?

I'd be happy to try and update it and make sure it runs on the latest stable version

MarkLyck commented:
@frinyvonnick I have updated the package, the tests, the docs and updated `pkg` to the new `packageJson` to be consistent in the code-base.

Let me know if there's anything else I missed.

MarkLyck commented:
@frinyvonnick did you have a chance to look at the PR?

frinyvonnick commented:
@MarkLyck I'll check it as soon as possible thank you for making the changes 👌

frinyvonnick commented:
It looks good to me @MarkLyck 👌

frinyvonnick commented:
This is available in the [canary version](https://docs.gitmoji-changelog.dev/#/?id=%f0%9f%90%a5-canary-version). I'll publish it as soon as possible 😄  Thank you for your contribution 🙏

frinyvonnick commented:
@all-contributors please add MarkLyck for code

frinyvonnick commented:
I was released in [2.2.1](https://github.com/frinyvonnick/gitmoji-changelog/releases/tag/v2.2.1) 🚀
